src/pages/donate/index.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
```

Comment:
Note that this file is already in the process of being converted in @candywang's PR, https://github.com/ShelterTechSF/sheltertech.org/pull/228.